model/registry/user-agent.yaml

```diff
@@ -0,0 +1,27 @@
+groups:
+  - id: registry.user_agent
+    prefix: user_agent
+    type: attribute_group
+    brief: "Describes user-agent attributes."
+    attributes:
+      - id: original
+        type: string
+        brief: >
+          Value of the [HTTP User-Agent](https://www.rfc-editor.org/rfc/rfc9110.html#field.user-agent) header sent by the client.
+        examples: ['CERN-LineMode/2.15 libwww/2.17b3',
+                   'Mozilla/5.0 (iPhone; CPU iPhone OS 14_7_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.1.2 Mobile/15E148 Safari/604.1']
+      - id: device_name
+        type: string
+        brief: >
+          Name of the device extracted from original
+        examples: ['iPhone']
+      - id: name
```

Comment:
What should this be in the event of multiple names?  User-Agent tends to be "additive" w/ layers of software, and isn't just browser-stacks.